backend/internal/controllers/middleware/utils.go

```diff
@@ -0,0 +1,12 @@
+package middleware
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// Sends data and status code for http
+func writeResponse(rw http.ResponseWriter, statusCode int, body interface{}) {
+	rw.WriteHeader(statusCode)
```

Comment:
You need to set the Content-Type header here, just like in `controllers.writeResponse`.